fdbserver/Knobs.cpp

```diff
@@ -682,7 +682,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( REDWOOD_LOGGING_INTERVAL,                              5.0 );
 
 	// Server request latency measurement
-	init( LATENCY_SAMPLE_SIZE,                                100000 );
+	init( LATENCY_SKETCH_ACCURACY,                               0.1 );
```

Comment:
Probably need to enumerate a set of legal accuracies that lead to power-of-two multipliers, or just round down to the next available value.

Also, I'm nervous about unbounded memory usage.  The integer variant bounds memory usage, but its range is only 1:2^32.  What if we created presets for latency, space, etc, and just used them everywhere?

Reply:
Regarding the unbounded memory, right now all numbers that are too small are considered as 0 and counted separately. All numbers that are too big raises an error (maybe we can treat this in a more elegant way). It should not use more than 1-2 thousand `double`s, which is basically as big as the previous `ContinuousSample` sketch.